### PR TITLE
Add NFTDescriptor's addresss on bytecode

### PR DESCRIPTION
**Pull request by AaronLee22**
Add NFTDescriptor's addresss on bytecode since it's a library contract.

### CODE REVIEW

File: op-chain-ops/genesis/layer_two.go [hunk 1]
```diff
@@ -140,6 +140,20 @@ func BuildL2Genesis(config *DeployConfig, l1StartBlock *types.Block) (*core.Gene
 				dep = c
 			}
 			deployResults[name] = dep
+		case "NFTDescriptor":
```
Comment (rlgns98kr) [inline op-chain-ops/genesis/layer_two.go:143]: Do you know which contract use this library? 🙇🏻‍♂️🙇🏻‍♂️🙇🏻‍♂️

Comment (xxeonge) [reply op-chain-ops/genesis/layer_two.go:143]: I think nonfungibleTokenPositionDescriptor.sol Is it right? @AaronLee22

**Review by rlgns98kr — APPROVED** [review]
Thank you very much! 🙇🏻‍♂️🙇🏻‍♂️🙇🏻‍♂️❤️❤️

File: op-chain-ops/genesis/layer_two.go (hunk 1 above)
Comment (AaronLee22) [reply op-chain-ops/genesis/layer_two.go:143]: It's NonfungibleTokenPositionDescriptor.sol.

**Pull request merged**